### PR TITLE
profile: CMake clean up

### PR DESCRIPTION
**Pull request by christophe-lunarg**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -108,23 +104,20 @@ if (VULKAN_HEADERS_INSTALL_DIR)
...
 find_package(valijson REQUIRED CONFIG)
 
 set(BUILD_SHARED_LIBS false)
 find_package(jsoncpp REQUIRED CONFIG)
-target_compile_options(jsoncpp_static INTERFACE "-fPIC")
 
 if(PROFILES_BUILD_TESTS)
-    set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
-    add_subdirectory(external/googletest)
+    #set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
```
Comment (juan-lunarg) [inline CMakeLists.txt:119]:
```suggestion
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -108,23 +104,20 @@ if (VULKAN_HEADERS_INSTALL_DIR)
 endif()
 
 find_package(VulkanHeaders REQUIRED CONFIG QUIET)
-set(VULKAN_HEADER_REGISTRY ${CMAKE_CURRENT_SOURCE_DIR}/external/Vulkan-Headers/registry)
+set(VULKAN_HEADER_REGISTRY ${VULKAN_HEADERS_INSTALL_DIR}/share/vulkan/registry)
```
Comment (juan-lunarg) [inline CMakeLists.txt:107]: Not needed. VulkanHeaders find_package support provides this for users:
https://github.com/KhronosGroup/Vulkan-Headers/blob/main/BUILD.md

```suggestion
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -108,23 +104,20 @@ if (VULKAN_HEADERS_INSTALL_DIR)
...
 
-include_directories(external/Vulkan-Headers/include)
 include_directories(${VULKAN_VALIDATIONLAYERS_INSTALL_DIR}/include/vulkan)
 set(Vulkan-ValidationLayers_INCLUDE_DIR "${VULKAN_VALIDATIONLAYERS_INSTALL_DIR}/include/vulkan")
 set(Vulkan-ValidationLayers_LIBRARY_DIR "${VULKAN_VALIDATIONLAYERS_INSTALL_DIR}/lib")
 
-
 find_package(valijson REQUIRED CONFIG)
 
 set(BUILD_SHARED_LIBS false)
```
Comment (juan-lunarg) [inline CMakeLists.txt:115]: No need to set BUILD_SHARED_LIB now that you aren't using FetchContent.

```suggestion
```

**Review by juan-lunarg — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (christophe-lunarg) [reply CMakeLists.txt]: This is resolved in the branch

**Review by juan-lunarg — APPROVED** [review]

**Review by juan-lunarg — APPROVED** [review]

**Pull request merged**